guide/additional-info/changes-in-v12.md

```diff
@@ -871,6 +1226,66 @@ The `RichEmbed` class has been removed in favor of the `MessageEmbed` class.
 + channel.createMessageCollector(filterFunction, { max: 2, maxProcessed: 10, time: 15000 });
 ```
 
+#### TextChannel#send\*\*\*
+
+All the `.send***()` methods have been removed in favor of one general `.send()` method.
+
+```diff
+- channel.sendMessage('Hey!');
++ channel.send('Hey!');
+```
+
+```diff
+- channel.sendEmbed(embedVariable);
++ channel.send(embedVariable);
++ channel.send({ embed: embedVariable });
+```
+
+<p class="warning">`channel.send(embedVariable)` will only work if that variable is an instance of the `MessageEmbed` class; object literals won't give you the expected result unless your embed data is inside an `embed` key.</p>
+
+```diff
+- channel.sendCode('js', 'const version = 11;');
++ channel.send('const version = 12;', { code: 'js' });
+```
+
+```diff
+- channel.sendFile('./file.png');
+- channel.sendFiles(['./file-one.png', './file-two.png']);
++ channel.send({
+  files: [{
+    attachment: 'entire/path/to/file.jpg',
+    name: 'file.jpg'
+  }]
++ channel.send({
+  files: ['https://cdn.discordapp.com/icons/222078108977594368/6e1019b3179d71046e463a75915e7244.png?size=2048']
```

Comment:
Missing a trailing comma

Reply:
Not adding trailing commas 'cause I don't like them and these examples are either legacy from when Sanc wrote them, or more likely since his stuff was out-of-date by a year, copied directly from the d.js docs and thus I'm following their style.